icubam/backoffice/templates/icu.html

```diff
@@ -20,100 +20,99 @@ <h2>{{_("Manage your")}} {{_("ICUs")}}</h2>
             </div>
 
-            <div class="card-footer">
-              <button type='button' onclick='navigateTo("{{list_route}}")'
-                class="btn btn-warning">{{_("Cancel")}}</button>
-              <button type="submit" class="btn btn-primary">{{_("Submit")}}</button>
+            <div class="col-20">
+              <label for="is_active">{{ _('Active') }}</label><br />
+              <input type="checkbox" name="is_active" {% if icu.is_active %} checked{% end %} data-bootstrap-switch
+                data-off-color="danger" data-on-color="success">
+            </div>
+          </div>
+
+          <div class="form-row">
+            <div class="col-25">
+              <label>{{ _('Region') }}</label>
+              <select class="form-control" name="region_id">
+                {% for region in regions %}
+                <option value="{{region.region_id}}" {% if icu.region_id == region.region_id %} selected {% end %}>
+                  {{region.name}}
+                  {% end %}
+              </select>
+            </div>
+            <div class="col-25">
+              <label for="city">{{ _('City') }}</label>
+              <input type="text" id="city" name="city" class="form-control" placeholder="{{ _('City') }}"
+                value="{% if icu.city %}{{icu.city}}{% end %}" required autofocus>
+            </div>
+
+            <div class="col-25">
+              <label for="dept">{{ _('Department / State') }}</label>
+              <input type="text" id="dept" name="dept" class="form-control" placeholder="{{ _('Department') }}"
+                value="{% if icu.dept %}{{icu.dept}}{% end %}" required autofocus>
+            </div>
+          </div>
+
+          <div class="form-row">
+            <div class="col-25">
+              <label for="lat">{{ _('Latitude') }}</label>
+              <input type="number" step="0.0001" id="lat" name="lat" class="form-control" placeholder="Latitude"
+                value="{%if icu.lat %}{{icu.lat}}{% end %}" required autofocus>
             </div>
-        </form>
-      </div>
-    </section>
 
-    {% if error %}
-    <div class="alert alert-danger" role="alert">
-      {{_("Failed to save ICU")}}
-    </div>
-    {% end %}
+            <div class="col-25">
+              <label for="long">{{ _('Longitude') }}</label>
+              <input type="number" step="0.0001" id="long" name="long" class="form-control"
+                placeholder="{{ _('Longitude') }}" value="{% if icu.long %}{{icu.long}}{% end %}" required autofocus>
+            </div>
+          </div>
+
+        </div>
 
+        <div class="card-footer">
+          <button type='button' onclick='navigateTo("{{list_route}}")' class="btn btn-warning">{{ _('Cancel') }}</button>
+          <button type="submit" class="btn btn-primary">{{ _('Submit') }}</button>
+        </div>
+      </form>
   </div>
 </section>
 
+{% if error %}
+<div class="alert alert-danger" role="alert">
+  {{_('Failed to save ICU')}}
```

Comment:
since you are on this. maybe we could save us some translations by having something like
`{{_('Error while saving ')}} {{_('ICU')}}`
It might not lead to perfect spanish or french, but it s only an error message, which appears elsewhere

Reply:
Sure, it goes in the same lines as the first comment. Will be updated.